sunpy/map/mapsequence.py

```diff
@@ -473,19 +471,38 @@ def peek(self, resample=None, **kwargs):
 
         return MapSequenceAnimator(plot_sequence, **kwargs)
 
+    @deprecated(since='6.1', message=v61_DEPRECATION_MESSAGE, alternative='data property')
```

Comment:
```suggestion
```

I don't think we should remove this, as it feels like a useful way to know if `.data` is going to fail and deprecating it without a replacement feels problematic.